cmd/limactl/edit.go

```diff
@@ -122,7 +122,7 @@ func editAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	return start.Start(ctx, inst)
+	return start.Start(ctx, inst, start.DefaultLaunchHostAgentForeground)
```

Comment:
Would it be better if the `--foreground` flag was also available for `limactl edit`?